apps/offlineevents/views.py

```diff
@@ -10,3 +19,102 @@ class OfflineEventDetailView(module_views.ItemDetailView):
     @property
     def project(self):
         return self.get_object().project
+
+
+class OfflineEventMgmtView(DashboardBaseMixin,
+                           rules_mixins.PermissionRequiredMixin,
+                           generic.ListView):
+    model = models.OfflineEvent
+    template_name = 'meinberlin_offlineevents/offlineevent_mgmt_list.html'
+    permission_required = 'a4projects.add_project'
+    menu_item = 'project'
+
+    def dispatch(self, *args, **kwargs):
+        self.project = Project.objects.get(slug=kwargs['slug'])
+        return super(OfflineEventMgmtView, self).dispatch(*args, **kwargs)
+
+    def get_queryset(self):
+        return super().get_queryset().filter(project__slug=self.project)
+
+
+class OfflineEventMgmtCreateView(rules_mixins.PermissionRequiredMixin,
+                                 generic.CreateView):
+
+    model = models.OfflineEvent
+    form_class = forms.OfflineEventForm
+    permission_required = 'meinberlin_offlineevents.add_offlineevent'
+    template_name = \
+        'meinberlin_offlineevents/offlineevent_mgmt_create_form.html'
+    menu_item = 'project'
+
+    @property
+    def organisation(self):
+        return self.project.organisation
+
+    def get_permission_object(self):
+        return self.organisation
```

Comment:
note: in topicprio a equivalent rule `meinberlin_topicprio.add_topic` is checked. there it does work, as `TopicMgmtCreateView` inherits from `ItemCreateView` which does lookup a module via the `slug` kwarg (which i deem also bad choice, as slug is just to gneric)